### DISCUSSION TITLE
请问拖拽注释到笔记，生成的格式用的是哪个模板

### DISCUSSION THREAD

**Discussion by Ward-zzZ** [windingwind/zotero-better-notes#803] [Q&A] [answered]
可以自定义这个格式吗
原注释：
<img width="159" alt="2023_11_13_16_05_18" src="https://github.com/windingwind/zotero-better-notes/assets/64120881/26f482bb-bd0a-4fbc-9c8a-03b6d5d870c0">
拖拽生成的笔记：
<img width="430" alt="2023_11_13_16_06_12" src="https://github.com/windingwind/zotero-better-notes/assets/64120881/e37782c8-cc4a-4419-ad55-e992aaf3e025">

**Comment by windingwind** [comment, marked as answer]
拖拽注释受zotero内部处理，不受插件控制。
可通过注释右上角的花心笔记图标点击后创建笔记，由QuickNote内置模板控制。

**Reply by Ward-zzZ** [reply]
噢噢好的，感谢
